coursework/product-design/index.md

```diff
@@ -340,36 +360,46 @@ A landing page cannot effectively convert visitors into leads/users if there are
+* Make promotional videos. View past CS3216 final project videos [here](https://www.youtube.com/playlist?list=PL-0mdQVieeceaFm__nZYSQYzhDGF0C9Jg).
+* [Griddit](https://www.youtube.com/watch?v=Jo_2aYXixqs&index=2&t=0s&list=PL-0mdQVieeceaFm__nZYSQYzhDGF0C9Jg), a live chat room app, got Prof Ben Leong to do an AMA using their service.
+* [Letterbox](https://www.youtube.com/watch?v=7TnnLeSPnhA&index=6&t=0s&list=PL-0mdQVieeceaFm__nZYSQYzhDGF0C9Jg) had a member going around the school on an Airwheel and giving out flyers.
 
 DO NOT:
 
-- Paste unremovable stickers on the benches of School of Computing.
-- Put up posters in inappropriate places like restrooms.
+* Paste unremovable stickers on the benches of the School of Computing.
+* Put up posters in inappropriate places like restrooms.
 
 Videos are not the only way to market an app - to obtain marks in the coveted 30%, think out of the box for non-disruptive, creative and effective marketing strategies. We want a well thought-out marketing plan including details on how you would implement it. Your strategy must obviously be realistic, i.e. do not suggest giving out $10 to each user who downloads your app if you do not have a good plan of getting those funds or a millionaire on your team.
 
 <div class="box">
   <strong class="milestone-counter">Milestone 11:</strong> Design a small marketing campaign for your app. Explain which marketing channels you would use and how you would use them to get the word out about your app to attract more users. Provide implementation details (e.g. if you decide to make a promotional video, write out a brief outline of the video contents). Explain why you think your marketing strategy would be effective in gathering users.
 </div>
 
+## Final Assignment Writeup
+
+Your submission will include a write-up that describes how your group has met all the milestones for this assignment:
+
+* It should contain answers to Milestones 1-4 and 10-11.
+* It should also sumamarise your Milestones 5-9. More specifically, it should:
+  * Make it clear how the designs are organised. Which ones are wireframes, mockups or prototypes?
+  * Briefly recap the results and observations from your user testing results.
+  * Highlight the main user flows presented in each draft / iteration submitted.
+
 ## Grading Scheme
 
-In view that this is a brand new assignment, the detailed grading scheme is still being worked out by the teaching staff. We ask for your kind understanding. It is possible that the scheme would evolve as we grade, but not to worry, this assignment is only worth 10% of the final grade and the goal is learn the Product Design process, so don't fret over the grading.
+The grading of the assignment is divided into two components: satisfying the compulsory milestones (70%) and coolness factor (30%). Excluding Milestone 0, there are **11 milestones** in total, along with an optional Milestone 8b. Milestones 6, 7 and Milestone 9 are each worth 10%. The rest are worth 5% each.
```

Comment:
@sooyj I have decided to take 5% off from Milestone 6 and give it to Milestone 7. Reasons are twofold:

- I feel that the students may be limited in what they can do for user testing for Milestone 6, especially now that it's likely to be done online. Things like body gestures and eye movements are now much harder to observe and document down, so 15% may be a bit too much.
- Furthermore, I feel that what's more important is how the documented results are utilised for design iteration. I am keen on seeing how the students analyse their results and use the results to justify and guide their subsequent design decisions. This thus comes under Milestone 7, which I have updated to ask for more detailed documentation from the students.

Overall, I feel the 3 x 10% milestones require relatively equal effort:

* User testing with 5 people VS
* Deriving insights and using that insights to drive changes in wireframe designs VS
* Converting the wireframe/mockup into an interactive prototype.

Would love to hear your thoughts on above.